### PR TITLE
11.0 mig account fiscal year closing

### PR DESCRIPTION
**Pull request by bodi000**
Added postgres to travis as checks were failing.
MIG of account_fiscalyear_closing to v11

### CODE REVIEW

File: account_fiscal_year_closing/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Fiscal year closing",
+    "summary": "Generic fiscal year closing wizard",
+    "version": "11.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://www.tecnativa.org/",
```
Comment (MeetKD) [inline account_fiscal_year_closing/__manifest__.py:10]: website -> https://github.com/OCA/account-closing/

File: account_fiscal_year_closing/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (MeetKD) [inline account_fiscal_year_closing/models/__init__.py:1]: Empty line can be removed and license terms ?

File: account_fiscal_year_closing/__manifest__.py (hunk 1 above)
Comment (bodi000) [reply account_fiscal_year_closing/__manifest__.py:10]: Thank you. I would ask @pedrobaeza what does he think, not a specialist in these matters myself.

File: account_fiscal_year_closing/models/__init__.py (hunk 2 above)
Comment (bodi000) [reply account_fiscal_year_closing/models/__init__.py:1]: Thank you. I would ask @pedrobaeza what does he think, not a specialist in these matters myself.

Comment (MeetKD) [reply account_fiscal_year_closing/models/__init__.py:1]: No Problem :+1:

File: account_fiscal_year_closing/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply account_fiscal_year_closing/__manifest__.py:10]: Yes, it's OK to change to GitHub OCA websites :)

**Comment by sergiocorato** [comment]
As in v. 12.0 this module is already ported, have you checked compatibility?

**Comment by matteoopenf** [comment]
it there problem to merge that pr?

**Comment by sergiocorato** [comment]
It would be better to check if the module in v. 12.0 has something different that might require a migration function, or check if this PR is equivalent to a backport from v. 12.0 so that the migration function is useless

**Comment by matteoopenf** [comment]
> It would be better to check if the module in v. 12.0 has something different that might require a migration function, or check if this PR is equivalent to a backport from v. 12.0 so that the migration function is useless

@bodi000  Can you help us  to understand the problem?

**Pull request closed without merging**